metashare/api/jobs.py

```diff
@@ -71,6 +76,47 @@ def _create_branches_on_github(*, user, repo_id, project, task):
     return task_branch_name
 
 
+def alert_user_about_expiring_org(*, org, days):
+    from .models import User, ScratchOrg
+
+    # if scratch org is there
+    try:
+        org.refresh_from_db()
+        user = org.owner
+        user.refresh_from_db()
+    except (ScratchOrg.DoesNotExist, User.DoesNotExist):
+        return
+
+    # and has unsaved changes
+    get_unsaved_changes(org)
+    if org.unsaved_changes:
+        domain = Site.objects.first().domain
+        task = org.task
+        project = task.project
+        repo = project.repository
+        path = f"/repositories/{repo.slug}/{project.slug}/{task.slug}/"
+        metashare_link = f"https://{domain}{path}"
```

Comment:
@wlonk I don't love this, but it was broken before -- displaying the API URL instead of the user-facing URL. I'd be happy for any cleanup you'd like to do. Some thoughts:

- This is the first time we're using the `Site` object, as far as I'm aware. If we're using it in this way, should we document that it needs to be set? Or also pay some attention to the `site.name`, instead of hardcoding "MetaShare" everywhere?
- This breaks if `site.domain` is set to e.g. "https://metashare-dev.herokuapp.com/", or even "metashare-dev.herokuapp.com/" (with a trailing slash) -- both of which are currently allowed. If we're going to use it this way, I think we need validation of its format.
- Should we also check for the value of `SECURE_SSL_REDIRECT` before assuming `https`?

Reply:
Oh, right, because while the frontend knows the API URLs, the backend doesn't know any frontend URLs.

 - We should document the use of `Site`, yes. I will.
 - We can assemble the URL via `purl` or similar to ensure it's well-formed no matter whether the `Site.domain` is entered correctly or not (the domain is supposed to be the _domain_, not a URL, so it shouldn't have a trailing slash, but that's not enforced/enforceable by the database).
 - That sounds reasonable, yeah, but I would prefer to use `https` everywhere except on `localhost`.

Reply:
- 👍
- 👍 If we're not validating domain, I think even just help text in the admin would be useful.
- 👍 Agreed -- but either we should always enforce https (and not allow overriding with an env var at all), or we should respect the env var setting.

Reply:
The admin is generated by the django Sites app, so modifying it would be rough, but I've put info into the docs. And validating logic.

I also changed the `https` logic a bit from what I said; it's now "if you're localhost _or_ if you're not `SECURE_SSL_REDIRECT`, use `http`."